src/coreclr/src/ToolBox/superpmi/superpmi/parallelsuperpmi.cpp

```diff
@@ -31,9 +31,18 @@ bool StartProcess(char* commandLine, HANDLE hStdOutput, HANDLE hStdError, HANDLE
 
     ZeroMemory(&pi, sizeof(pi));
 
+#ifdef TARGET_UNIX
```

Comment:
There seems to be number of cases where `CreateProcessA` is used and switching to CreateProcessW adds extra code. Is removing `CreateProcessA` really an improvement? I think I would keep it.

Reply:
This is the only instance in the whole repo.

Reply:
ok, sounds good.